client/src/components/Applications/AppForm/AppForm.js

```diff
@@ -61,9 +67,9 @@ const AppDetail = () => {
     const errorText = error.message ? error.message : 'Something went wrong.';
 
     return (
-      <Message type="warn" className="appDetail-error">
+      <SectionMessage type="warning" className="appDetail-error">
         {errorText}
-      </Message>
+      </SectionMessage>
```

Comment:
@tacc-wbomar  Is this supposed to be full-height?
<img width="1785" alt="Screen Shot 2020-12-16 at 2 01 50 PM" src="https://user-images.githubusercontent.com/20326896/102400285-50a4d480-3fa7-11eb-863c-b2633946a957.png">

Reply:
No. The `<SectionMessage>` should be an element _within_ the `class="appDetail-error"`.

I have found working with `appDetail-error` to be obtuse. Now that I remember its varied application, I revise my comment on another instance. See https://github.com/TACC/Frontera-Portal/pull/228#discussion_r545229487.